ee/clickhouse/client.py

```diff
@@ -207,24 +218,20 @@ def _notify_of_slow_query_failure(tags: Dict[str, Any]):
     incr("clickhouse_sync_execution_failure", tags=tags)
 
 
-def format_sql(sql, params, colorize=True):
-    substitute_params = (
-        ch_client.substitute_params if isinstance(ch_client, SyncClient) else ch_client._client.substitute_params
```

Comment:
Was this if statement dead previously? New code doesn't do this.

Reply:
`get_client()` appears to always return the same thing as far as I can tell. `ch_client` is variable. Although maybe I'm missing something 🤔